.eslintrc.js

```diff
@@ -25,6 +25,23 @@ module.exports = {
     "@typescript-eslint/no-misused-promises": "off",
     "@typescript-eslint/no-base-to-string": "off",
 
+    // Warn
```

Comment:
some warnings to make sure jsdocs are kept formatted correct

Reply:
+1